test/models/return-cycle.model.test.js

```diff
@@ -50,8 +50,12 @@ describe('Return Cycle model', () => {
         expect(result).to.be.instanceOf(ReturnCycleModel)
         expect(result.id).to.equal(testRecord.id)
 
-        expect(result.returnLogs[0]).to.be.an.instanceOf(ReturnLogModel)
-        expect(result.returnLogs[0]).to.equal(testReturnLog)
+        const expectedReturnLog = result.returnLogs.find((returnLog) => {
+          return returnLog.id === testReturnLog.id
+        })
+
+        expect(expectedReturnLog).to.be.an.instanceOf(ReturnLogModel)
+        expect(expectedReturnLog).to.equal(testReturnLog)
```

Comment:
I can see why this was done. But when viewing the file as a whole to confirm it, I spotted we've chosen the wrong style of test.

The pattern we've followed is for a 'BelongsTo' or 'HasOne' relationship, when we should have followed the pattern for a 'HasMany'.

So, this would be a suitable time for a bit of housekeeping to correct the test pattern. I then also wonder if you could get the expect to work without needing the `find()` call by switching it to be `expect(result.returnlogs).to.include(testReturnLogs[0])`. We've done this in other model tests and it has worked fine.